tests/foreman/api/test_contentcredentials.py

```diff
@@ -227,3 +229,46 @@ def test_positive_delete(module_org):
     gpg_key.delete()
     with pytest.raises(HTTPError):
         gpg_key.read()
+
+
+@pytest.mark.tier1
```

Comment:
I would maybe argue that these are `tier3` tests, but I'm not the best at labeling this marker. What do others think? https://github.com/SatelliteQE/robottelo/blob/master/pytest_plugins/markers.py#L11 for context on tiers

Reply:
Changed to `tier3` for now, also would like to hear what others think about this one